Wrappers/Python/cil/optimisation/operators/Operator.py

```diff
@@ -129,25 +135,42 @@ class LinearOperator(Operator):
     range_geometry : ImageGeometry or AcquisitionGeometry, optional, default None
         range of the operator 
     """
+
     def __init__(self, domain_geometry, **kwargs):
         super(LinearOperator, self).__init__(domain_geometry, **kwargs)
+
     def is_linear(self):
         '''Returns if the operator is linear'''
         return True
-    def adjoint(self,x, out=None):
+
+    def adjoint(self, x, out=None):
         '''returns the adjoint/inverse operation
-        
+
         only available to linear operators'''
         raise NotImplementedError
-    
-    @staticmethod
-    def PowerMethod(operator, max_iteration=10, initial=None, tolerance = 1e-5,  return_all=False):
 
+    @staticmethod
+    def PowerMethod(operator, max_iteration=10, initial=None, tolerance=1e-5,  return_all=False, method='auto'):
         r"""Power method or Power iteration algorithm 
-        
-        The Power method computes the largest (dominant) eigenvalue of a square matrix in magnitude, e.g.,
-        absolute value in the real case and module in the complex case.        
-        For the non-square case, the power method is applied for the matrix :math: A^{T}*A.
+
+        The power method contains two different algorithms chosen by the `method` flag. 

```

Comment:
I think the doc string should just have a short description initially so the parameters are near the top. In the parameters section the `method` argument just has a default, and in the description list the allowed types and a very short sentence i.e. "`auto` lets the code choose the method, this can be specified with `direct_only` or `compose_with_adjoint`" 

Then futher explanation of each method can be in a 'Note' or 'Notes' section (both exist but I forget which is the most apporiate... have a look at the PDHG doc string.